dev/droidmate/projects/core/src/main/groovy/org/droidmate/android_sdk/AdbWrapper.groovy

```diff
@@ -610,9 +619,15 @@ public class AdbWrapper implements IAdbWrapper
 
     try
     {
-      sysCmdExecutor.execute(commandDescription, cfg.adbCommand,
-        "-s", deviceSerialNumber,
-        "shell rm", filePath)
+      if (runAsPackage == null)
```

Comment:
If I am correct, in current code this is never true. Thus, assume package is always non-null. I.e. add at the beginning of method: assert runAsPackage != null